thorvald_001:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: thorvald_001_arm_joint1
    pid: {p: 80, i: 0.2, d: 6}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: thorvald_001_arm_joint2
    pid: {p: 40, i: 0.2, d: 6.0}
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: thorvald_001_arm_nozzle_joint
    pid: {p: 10.0, i: 0.01, d: 1.0}
